docs/v3/source/includes/api_resources/_security_groups.erb

```diff
@@ -41,12 +41,12 @@
 }
 <% end %>
 
-<% content_for :paginated_security_group, File.read(__dir__ + '/examples/security_group_paginated.json') %>
+<% content_for :paginated_security_group, File.read('source/includes/api_resources/examples/security_group_paginated.json') %>
```

Comment:
These were probably the weirdest change needed. It seems like the rendering context for templates is different in middleman 4, so `__dir__` doesn't return the correct path.